float32 x_position
float32 y_position
float32 x_velocity
float32 y_velocity
---
Header header
float32 x
float32 y
float32 theta

float32 x_velocity
float32 y_velocity
float32 theta_velocity